scripts/install_stacks.sh

```diff
@@ -50,20 +49,44 @@ BTC_ADDRESS=$(sudo cat  /root/keychain.json | jq .keyInfo.btcAddress | tr -d '"'
 STX_ADDRESS=$(sudo cat  /root/keychain.json | jq .keyInfo.address | tr -d '"')
 
 if [ ! -f "/bitcoin/wallet.dat" ]; then
+    echo "[ install_stacks.sh ] - Creating bitcoin wallet"
+    bitcoin-cli \
+        -rpcconnect=127.0.0.1 \
+        -rpcport=8332 \
+        -rpcuser=btcuser \
+        -rpcpassword=btcpass \
+        createwallet "miner" \
+        false \
+        false \
+        "" \
+        false \
+        false \
+        true
+
+    echo "[ install_stacks.sh ] - Restarting bitcoin"
+    sudo systemctl restart bitcoin
     echo "[ install_stacks.sh ] - Sleeping for 120 seconds to allow bitcoin time to restart"
     sleep 120
 
+    echo "[ install_stacks.sh ] - Loading bitcoin wallet"
+    bitcoin-cli \
+        -rpcconnect=127.0.0.1 \
+        -rpcport=8332 \
+        -rpcuser=btcuser \
+        -rpcpassword=btcpass \
+        loadwallet miner
```

Comment:
no longer required
```suggestion
```

Reply:
Did you manage to test it without? I've added it because after restarting the bitcoin node, the wallet is not automatically imported (perhaps we can add that feature and remove this code, but only removing it might not work). Last time I tested, I could not import the address because the wallet wasn't loaded, that's the reason of this addition.